examples/using-preact/package.json

```diff
@@ -14,7 +14,9 @@
     "preact-render-to-string": "^5.1.11",
     "react": "npm:@preact/compat",
     "react-dom": "npm:@preact/compat",
-    "react-ssr-prepass": "npm:preact-ssr-prepass@^1.1.2"
+    "react-ssr-prepass": "npm:preact-ssr-prepass@^1.1.2",
+    "webpack": "^5.4.0",
```

Comment:
This means there will be 2 versions of webpack running which is not ideal (and will cause failures later). The root cause should be fixed.

Reply:
Do you have any pointers as to where the root cause might be? Is it an error with preact or the next.js example?

Reply:
Next.js bundles webpack, it seems that `next-plugin-preact` is using webpack internals that can't be required as they're no longer available, so that'd be a starting point.

Reply:
https://github.com/preactjs/prefresh/blob/main/packages/webpack/src/index.js#L60-L62